packages/preview/anr-aapg-fanmade/0.1.0/typst.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "anr-aapg-fanmade"
+version = "0.1.0"
+entrypoint = "aapg.typ"
+authors = ["Gabriel Scherer"]
+license = "ISC"
+description = "Typst templates for ANR AAPG: grant proposals for the French National Research Agency"
+
+repository = "https://gitlab.com/gasche/anr-aapg-fanmade-typst-template/"
+categories = ["report"]
+
+## Typst only supports one template per package,
+## see https://github.com/typst/packages/issues/1059
+# [template]
+# path = "template"
+# entrypoint = "first-proposal.typ"
+# thumbnail = "first-proposal-thumbnail.png"
+
+[template]
+path = "template"
+entrypoint = "second-proposal.typ"
+thumbnail = "second-proposal-thumbnail.png"
```

Comment:
Thumbnail files should be outside of the template directory: this directory only contains files that will be copied to the user's project. The `thumbnail` key in the manifest is relative to the root of the package, not to the template directory.

Reply:
Thanks! I also changed the thumbnails from .png to .webp, following the discussion in https://github.com/typst/packages/pull/2007 .